subsys/secure_storage/compatibility/Kconfig

```
@@ -0,0 +1,79 @@
+# Copyright (c) 2025 Nordic Semiconductor
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+config SECURE_STORAGE_TRUSTED_STORAGE_COMPATIBILITY
+	bool "Trusted storage backward compatibility [EXPERIMENTAL]"
+	depends on SECURE_STORAGE_ITS_STORE_IMPLEMENTATION_SETTINGS || \
+		   SECURE_STORAGE_ITS_STORE_IMPLEMENTATION_CUSTOM
+	depends on (SETTINGS_ZMS || SETTINGS_ZMS_LEGACY || \
+		    (SETTINGS_NVS && !SOC_SERIES_NRF54LX)) || \
+		   SECURE_STORAGE_ITS_STORE_IMPLEMENTATION_CUSTOM
+	select EXPERIMENTAL
+	select SECURE_STORAGE_ITS_STORE_SETTINGS_NAME_CUSTOM \
+	       if SECURE_STORAGE_ITS_STORE_IMPLEMENTATION_SETTINGS
+	help
+	  Enable this to make the secure storage subsystem compatible with
+	  an existing installation that was previously using the trusted storage library.
+	  It will allow the secure storage subsystem to operate and store entries
+	  like the trusted storage library would.
+
+# Replace the upstream AEAD implementation by a Trusted Storage Backward Compatible (TSBC) one.
+# The custom option (SECURE_STORAGE_ITS_TRANSFORM_IMPLEMENTATION_CUSTOM) remains usable.
+choice SECURE_STORAGE_ITS_TRANSFORM_IMPLEMENTATION
+	default SECURE_STORAGE_ITS_TRANSFORM_IMPLEMENTATION_TSBC
+
+config SECURE_STORAGE_ITS_TRANSFORM_IMPLEMENTATION_TSBC
+	bool "ITS transform module implementation compatible with trusted storage"
```

```suggestion
	bool "ITS transform module implementation compatible with trusted storage"
```
Do you mean ITS with "trusted storage" or the trusted storage library?
Maybe:
```suggestion
	bool "Implementation of the ITS transform module, compatible with the Trusted storage library"
```